src/logger.service.ts

```diff
@@ -82,7 +81,7 @@ export class NGXLogger {
     }
   }
 
-  private _log(level: NgxLoggerLevel, message, additional: any[] = [], logOnServer: boolean = true): void {
+  private _log(level: NgxLoggerLevel, message, additional: any[] = [], saveLog = true): void {
```

Comment:
save log still feels too non-descriptive to me. Can you explain why you think this is better than logOnServer?

Reply:
This kind of builds from my comment from earlier. I don't want people to assume they have to log it to the server. There is many different approaches a developer might take to storing the log and logOnServer is too tied to a specific way of saving a log. One can still do what they need by overriding logOnServer but it will be semantically off when a developer goes to implement it for other reasons.